README.md

```diff
@@ -16,63 +16,64 @@ The following projects are official OpenJS Foundation projects. If you are inter
+| <img alt="logo of project jQuery UI" src="https://github.com/openjs-foundation/artwork/raw/main/projects/jquery_ui/jquery_ui-icon-color.svg" width="30" /> | [jQuery UI](https://jqueryui.com/)                |    | [Contributing Guide](https://github.com/jquery/jquery-ui/blob/master/CONTRIBUTING.md) | [Code of Conduct](https://github.com/openjs-foundation/cross-project-council/blob/master/CODE_OF_CONDUCT.md) |
+| <img alt="logo of project RequireJS" src="https://github.com/openjs-foundation/artwork/raw/main/projects/requirejs/requirejs-logo-stacked-color.svg" width="30" /> | [RequireJS](https://requirejs.org/)               |    | [Contributing Guide](https://requirejs.org/docs/contributing.html) | [Code of Conduct](https://github.com/openjs-foundation/cross-project-council/blob/master/CODE_OF_CONDUCT.md) |
+| <img alt="logo of project Sizzle" src="https://github.com/openjs-foundation/artwork/raw/main/projects/sizzle/sizzle-icon-color.svg" width="30" /> | [Sizzle](https://sizzlejs.com/)                   |    | [Contributing Guide](https://github.com/jquery/sizzle/blob/master/CONTRIBUTING.md) | [Code of Conduct](https://github.com/openjs-foundation/cross-project-council/blob/master/CODE_OF_CONDUCT.md) |
 
 #### Incubating Projects
 
 | | Project        | Charter           | Contributing       | Code of Conduct       |
 |-|----------------|:-----------------:|:------------------:|:---------------------:|
-| | [NativeScript](https://nativescript.org/) |   | [Contributing Guide](https://github.com/NativeScript/NativeScript/blob/master/tools/notes/CONTRIBUTING.md) | [Code of Conduct](https://github.com/NativeScript/codeofconduct) |
-| | [nvm](http://nvm.sh/) |   | [Contributing Guide](https://github.com/nvm-sh/nvm/blob/master/CONTRIBUTING.md) |  |
+| <img alt="logo of project ajv" src="https://github.com/openjs-foundation/artwork/raw/main/projects/ajv/ajv-logo-color.svg" width="30" />  | [Ajv](https://ajv.js.org/) |   | [Contributing Guide](https://ajv.js.org/contribute.html) | [Code of Conduct](https://github.com/ajv-validator/ajv/blob/master/CODE_OF_CONDUCT.md) |
+| <img alt="logo of project NativeScript" src="https://github.com/openjs-foundation/artwork/raw/main/projects/nativescript/nativescript-icon-color.svg" width="30" /> | [NativeScript](https://nativescript.org/) |   | [Contributing Guide](https://github.com/NativeScript/NativeScript/blob/master/tools/notes/CONTRIBUTING.md) | [Code of Conduct](https://github.com/NativeScript/codeofconduct) |
+| <img alt="logo of project nvm" src="https://github.com/openjs-foundation/artwork/raw/main/projects/nvm/nvm-logo-grayscale.svg" width="30" />  | [nvm](http://nvm.sh/) |   | [Contributing Guide](https://github.com/nvm-sh/nvm/blob/master/CONTRIBUTING.md) |  |
```

Comment:
lol i guess this is nvm's logo now?
```suggestion
| <img alt="logo of project nvm" src="https://github.com/openjs-foundation/artwork/raw/main/projects/nvm/nvm-logo-grayscale.svg" width="30" />  | [nvm](http://nvm.sh/) |   | [Contributing Guide](https://github.com/nvm-sh/nvm/blob/HEAD/CONTRIBUTING.md) |  |
```

Reply:
It's the one you and I discussed a while back, yeah :-)

Reply:
If there's another one to use, we can definitely sub it out. Just seemed better than leaving it empty...

Reply:
nah this is fine, i just didn't remember where it came from

Reply:
Alright cool. I guess it was a placeholder that stayed a placeholder. Easy enough to swap at any point though.